### ISSUE TITLE
New update of django-modelcluster to 2.0 breaks wagtail v1.0b1

### ISSUE THREAD

**Issue by salvadormrf** [torchbox/wagtail#2506]
```
Traceback (most recent call last):
  File "manage.py", line 23, in <module>
    execute_from_command_line(sys.argv)
  File "vendor/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "vendor/django/core/management/__init__.py", line 377, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "vendor/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "vendor/django/core/management/base.py", line 337, in execute
    self.check()
  File "vendor/django/core/management/base.py", line 371, in check
    all_issues = checks.run_checks(app_configs=app_configs, tags=tags)
  File "vendor/django/core/checks/registry.py", line 59, in run_checks
    new_errors = check(app_configs=app_configs)
  File "vendor/django/core/checks/model_checks.py", line 28, in check_all_models
    errors.extend(model.check(**kwargs))
  File "vendor/django/db/models/base.py", line 1046, in check
    errors.extend(cls._check_fields(**kwargs))
  File "vendor/django/db/models/base.py", line 1122, in _check_fields
    errors.extend(field.check(**kwargs))
  File "/project/vendor/modelcluster/fields.py", line 278, in check
    if self.rel.get_accessor_name() == '+':
AttributeError: 'ManyToOneRel' object has no attribute 'get_accessor_name'
```
Since I am using an older version of wagtail, almost every time there is new dependency update it breaks the wagtail site. To fix this issue, we install the working versions just after wagtail dependency.

```
...
wagtail==1.0b1
django-libsass==0.6
django-compressor==1.6
django-modelcluster==1.1
django-medusa==0.3.0
Django==1.7.7
...
```

**Comment by auvipy** [comment]
use wagtail 1.4+ version. is it wise to use unsupported version of django? u should use minimum 1.8 as its LTS. beside why using beta version of wagtail?

**Comment by gasman** [comment]
Since Wagtail 1.4 we've made our dependency version requirements stricter:
https://github.com/torchbox/wagtail/blob/stable/1.0.x/setup.py#L26-L37
vs
https://github.com/torchbox/wagtail/blob/stable/1.4.x/setup.py#L24-L33

As such, this issue has already been fixed - please upgrade to Wagtail 1.4.
